composer.json

```
@@ -10,24 +10,23 @@
         }
     ],
     "require": {
-        "php": "^7.2|^8.0",
+        "php": "^8.1",
         "ext-dom": "*",
-        "ext-json": "*",
-        "illuminate/contracts": "^7.0|^8.0|^9.0|^10.0",
-        "illuminate/database": "^7.0|^8.0|^9.0|^10.0",
-        "illuminate/http": "^7.0|^8.0|^9.0|^10.0",
-        "illuminate/support": "^7.0|^8.0|^9.0|^10.0",
-        "illuminate/testing": "^7.0|^8.0|^9.0|^10.0",
+        "illuminate/contracts": "^10.0",
+        "illuminate/database": "^10.0",
+        "illuminate/http": "^10.0",
+        "illuminate/support": "^10.0",
+        "illuminate/testing": "^10.0",
         "mockery/mockery": "^1.0",
-        "phpunit/phpunit": "^8.5|^9.0",
-        "symfony/console": "^5.0|^6.0",
-        "symfony/css-selector": "^5.0|^6.0",
-        "symfony/dom-crawler": "^5.0|^6.0",
-        "symfony/http-foundation": "^5.0|^6.0",
-        "symfony/http-kernel": "^5.0|^6.0"
+        "phpunit/phpunit": "^10.0",
```

```suggestion
        "phpunit/phpunit": "^10.0.7",
```

Match Laravel Framework 10.